### DISCUSSION TITLE
Cancel listing on auction house

### DISCUSSION THREAD

**Discussion by johnsonchau-bulb** [metaplex-foundation/js#221] [Q&A] [answered]
Hi all,

For the auction house module, I was wondering how does one cancel a listing.

[createListing.ts](https://github.com/metaplex-foundation/js/blob/main/packages/js/src/plugins/auctionHouseModule/createListing.ts) allows users to create a listing but there should also be a way to cancel listing.

Thanks!

**Comment by lorisleiva** [comment, marked as answer]
Hi there 👋

This is possible using the `cancelListing` operation, but please remember that the Auction House module is not yet complete. We're still working on it and breaking changes might frequently occur as the module has not been officially released on this SDK. Thank you for your patience.
